test/muFundsFunction.js

```diff
@@ -0,0 +1,92 @@
+const record = false;
```

Comment:
@joseballester I decided to add start of a test suite as well. Would you be up for having something like this in there? I can add more to this if you think this is a good direction to go in.